Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2.yml

```diff
@@ -1044,6 +1044,13 @@ script:
       isArray: true
     - description: The email address from which to reply.
       name: from
+    - description: Whether to handle inline images in the HTML body. When set to 'True', inline images will be extracted from the HTML body and will be attached to the email as an inline attachment object.
```

Comment:
- description: Whether to handle inline images in the HTML body. When set to 'True', inline images are extracted from the HTML body and attached to the email as an inline attachment object.